ops-bedrock/devnet-up.sh

```diff
@@ -69,7 +69,7 @@ if [ ! -f ./.devnet/genesis-l1.json ]; then
   echo "Regenerating L1 genesis."
   (
     cd $CONTRACTS_BEDROCK
-    L1_GENESIS_TIMESTAMP=$L1_GENESIS_TIMESTAMP npx hardhat genesis-l1 \
+    L1_GENESIS_TIMESTAMP=$L1_GENESIS_TIMESTAMP npx hardhat --network $NETWORK genesis-l1 \
```

Comment:
Passing through the env var here can be removed and we can use the flag instead

Reply:
Don't even need to use the flag. It'll generate a timestamp as part of the task. And if the l1 genesis already exists, we don't run the genesis l1 task anyway, so no need to replicate the timestamp.